src/shared/components/common/pictrs-image.tsx

```diff
@@ -22,7 +22,7 @@ export class PictrsImage extends Component<PictrsImageProps, any> {
 
   render() {
     return (
-      <picture className="d-inline-block overflow-hidden">
+      <picture className="pictrs-image d-inline-block overflow-hidden">
```

Comment:
This `picture` element should be easily queried, no? Why does it need an additional class? This is the only use of a `picture` element.

Reply:
It may not remain that so.  It's contextualized to the component, which is the convention I used.